src/js/components/ContainerSettingsComponent.jsx

```diff
@@ -78,7 +78,7 @@ var ContainerSettingsComponent = React.createClass({
         <div className="col-sm-3">
           <FormGroupComponent
             attribute={`container.docker.portMappings[${i}].containerPort`}
-            label="Container Port"
+            label="Container port"
```

Comment:
OK, sentence case it is. Not sure what the typographical/layout advantage is though.